BeerOverflowWindowsApp/GetBarList.cs

```diff
@@ -24,6 +24,7 @@ private void Go_Click(object sender, EventArgs e)
                 GetBarListFourSquare barListFourSquare = new GetBarListFourSquare();
                 var result = barListGoogle.GetBarsAround(GetLatitude(), GetLongitude(), GetRadius());
                 result = CombineLists(result, barListFourSquare.GetBarsAround(GetLatitude(), GetLongitude(), GetRadius()));
+                result = CombineLists(result, new GetBarListTripAdvisor().GetBarsAround(GetLatitude(),GetLongitude(),GetRadius()));
```

Comment:
This is a really bad way to do this. I think just make CombineLists take more than 2 lists, use params for that.